open-company/open-company-web
# Force trasparent background color for links in the body.

Card: https://trello.com/c/BHVgwfDy

To test:
- add a new post
- leave empty abstract
- add some text and a link in the body (make sure the link is on the first or second line)
- post it
- [x] do you see the link with transparent background?
- [x] do you see the link with transparent background when you hover the post?
